### PR TITLE
Fix jp translation

### PR DESCRIPTION
**Pull request by yshrsmz**
- [x] I did these translations myself and own copyright for them
- [x] I didn't use a machine to do these translations
- [x] I assign all copyright to Loom Network for these translations

### CODE REVIEW

File: jp/4/battle-02.md [hunk 1]
```diff
@@ -252,7 +252,7 @@ uint random2 = uint(keccak256(now, msg.sender, randNonce)) % 100;
 
 例えばコイン・トスのコントラクトがあるとしよう。表ならお主のお金は２倍になり、裏なら全てのお金を失うというルールだ。表裏を決定するのに、上でやった乱数関数を使うとしよう。(`random >= 50`なら表、`random < 50`なら裏)
 
-もしわしがノードを立てているとしたら、 **自分のノードだけに向けて** トランザクションを発行できる。そうするとコイン・トスの関数を動かして勝ち負けを見て、わしの解いている次のブロックにそのトランザクションが含まれないよう選択することができてしまう。コイン・トスに勝って次のブロックを解き、利益を出すまでこれを無期限で行うことが可能だ。
+もしわしがノードを立てているとしたら、 **自分のノードだけに向けて** トランザクションを発行できる。そうするとコイン・トスの関数を動かして勝ち負けを見て、わしの解いている次のブロックにそのトランザクションが含まれないよう選択することができてしまう。わしはこれをコイン・トスに勝つまで無制限に続けることができ、勝ったらこのトランザクションを含めた次のブロックを解き、利益を得ることが可能だ。
```
Comment (yshrsmz) [inline jp/4/battle-02.md]: > I could keep doing this indefinitely until I finally won the coin flip and solved the next block, and profit.

原文はこれですが、 `indefinitely...` は文全体にかかるのではなくコイン・トスだけにかかるのだと思います。

Comment (mamiM) [reply jp/4/battle-02.md]: コイン・トスに勝って次のブロックを解くまでこれを無期限で行い続け、利益を出すことが可能だ。

でお願いします。

I could *keep* doing this indefinitely until I finally **won** the coin flip and **solved** the next block, and *profit*.

同じ印の動詞が並列の関係なのでindefinitelyは**の動詞にかかっているようです。

**Comment by yshrsmz** [comment]
@mamiM thanks for your review! updated.

**Comment by eduardonunesp** [comment]
Thanks @yshrsmz @mamiM

**Pull request merged**